opus/application/apps/ui/templates/ui/base.html

```diff
@@ -638,6 +638,9 @@ <h3 class='op-header-text'></h3>
     <script src="https://code.jquery.com/jquery-3.3.1.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
     <script>window.jQuery || document.write('<script src="{{ STATIC_URL }}cdn_fallback/jquery/js/jquery-3.3.1.min.js"><\/script>')</script>
 
+    <!-- flexslider -->
+    <script src="{{ STATIC_URL }}flexslider/jquery.flexslider.js"></script>
```

Comment:
why did we decide not to make use of a CDN copy of flexslider?

Reply:
Updated to use CDN for flexslider now. Local files have been moved to cdn_fallback folder.